src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderApprovedDocumentServiceTest.java

```diff
@@ -2,66 +2,163 @@
 import static uk.gov.hmcts.reform.finrem.caseorchestration.SetUpUtils.pensionDocumentData;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.TEST_SOLICITOR_NAME;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.TEST_SOLICITOR_REFERENCE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APPLICANT_REPRESENTED;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APP_ADDRESS_CCD_FIELD;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APP_FIRST_AND_MIDDLE_NAME_CCD_FIELD;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APP_LAST_NAME_CCD_FIELD;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APP_RESP_FIRST_AND_MIDDLE_NAME_CCD_FIELD;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APP_RESP_LAST_NAME_CCD_FIELD;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APP_SOLICITOR_ADDRESS_CCD_FIELD;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONSENT_ORDER_APPROVED_NOTIFICATION_LETTER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.SOLICITOR_NAME;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.SOLICITOR_REFERENCE;
 
 public class ConsentOrderApprovedDocumentServiceTest {
 
     @Mock
     private DocumentClient documentClientMock;
 
-    private DocumentConfiguration config;
-
     private ObjectMapper mapper = new ObjectMapper();
-
     private ConsentOrderApprovedDocumentService service;
+    private LetterAddressHelper letterAddressHelper = new LetterAddressHelper();
+    private CaseDetails caseDetails;
 
     @Before
     public void setUp() {
-        config = new DocumentConfiguration();
-        config.setApprovedConsentOrderTemplate("test_template");
-        config.setApprovedConsentOrderFileName("test_file");
+        DocumentConfiguration config = new DocumentConfiguration();
+        config.setApprovedConsentOrderTemplate("FL-FRM-DEC-ENG-00071.docx");
+        config.setApprovedConsentOrderFileName("ApprovedConsentOrderLetter.pdf");
+        config.setApprovedConsentOrderNotificationTemplate("FL-FRM-LET-ENG-00095.docx");
+        config.setApprovedConsentOrderNotificationFileName("ApprovedConsentOrderNotificationLetter.pdf");
         documentClientMock = mock(DocumentClient.class);
-        service = new ConsentOrderApprovedDocumentService(documentClientMock, config, mapper);
+        service = new ConsentOrderApprovedDocumentService(documentClientMock, config, mapper, letterAddressHelper);
+
+        Map<String, Object> applicantAddress = new HashMap<>();
```

Comment:
Why don't we use `Address` class?

Maps are a cancer of our code :( [it's not about you code, but ccd-like solution :( ]